common/src/main/scala/uk/ac/wellcome/models/MiroTransformable.scala

```diff
@@ -23,9 +23,19 @@ case class MiroTransformableData(
   @JsonProperty("image_keywords_unauth") keywordsUnauth: Option[List[String]],
   @JsonProperty("image_phys_format") physFormat: Option[String],
   @JsonProperty("image_lc_genre") lcGenre: Option[String],
-  @JsonProperty("image_tech_file_size") techFileSize: Option[List[String]]
+  @JsonProperty("image_tech_file_size") techFileSize: Option[List[String]],
+  @JsonProperty("image_use_restrictions") useRestrictions: Option[String]
 )
 
+/** Thrown for errors in the data which need further investigation or indicate
+ *  a transformer bug.
+ */
+case class MiroDataException(message: String)
```

Comment:
Be aware this will trigger the try backoff failure state which may not be what you intended - a ShouldNotTransformException will not do so (see `SQSMessageReceiver`).